docs/whiteblock-test-plan.md

```diff
@@ -66,49 +63,48 @@ series focuses on observing and documenting the effects of certain conditions
 on performance. Each test series is comprised of three separate test cases
 which define the variable to be tested. 
 
-### Series 1: Number of Validators
+### Series 1: Validators
 
 | Variable         | Test Case A | Test Case B | Test Case C |
 |------------------|------------:|------------:|------------:|
-| Validators       | 300         | 600         | 1000        |
-| Static Nodes     | 600         | 600         | 600         |
+| Validators       | 10          | 20          | 30          |
+| Static Nodes     | 6           | 6           | 6           |
 | Contract         | dupe.rho    | dupe.rho    | dupe.rho    |
-| Bandwidth        | 5Mb         | 5Mb         | 5Mb         |
-| Propagation Time | 8s          | 8s          | 8s          |
-
+| Bandwidth        | 1Gb         | 1Gb         | 1Gb         |
+| Network Latency  | 0ms         | 0ms         | 0ms         |
+| Packet Loss      | 0%          | 0%          | 0%          |
 
 ### Series 2: Number of Static Nodes
 
 | Variable        | Test Case A | Test Case B | Test Case C |
 |-----------------|------------:|------------:|------------:|
-| Validators      | 600         | 600         | 600         |
-| Static Nodes    | 300         | 600         | 1000        |
+| Validators      | 6           | 5           | 5           |
+| Static Nodes    | 10          | 20          | 30          |
```

Comment:
What's a static node? Can't find anything on that in the wiki

Reply:
A passive node that isn't acting as a validator. Could be a node that's sending or receiving transactions, deploying contracts, etc.

Reply:
This is a bit confusing. My first thought was that this is a new name for a read only node. See: https://rchain.atlassian.net/wiki/spaces/CORE/pages/495747126/Read+Only+Nodes
I think the words static/passive should be reserved for those kinds of nodes.

Perhaps the nodes you described should be called "standard" to avoid confusion?

Reply:
The description in that link is a bit vague. Can Read Only Nodes still deploy smart contracts and conduct transactions?

Reply:
Ok, then I think the term static node is more appropriate and general since they can be filling any other role that isn't specific to validators.